e2e-tests/content-strings/fields/insurance/your-buyer/index.js

```diff
@@ -4,5 +4,9 @@ export const FIELDS = {
       ID: 'country',
       LABEL: 'Country',
     },
+    BUYER_ORGANISATION: {
+      ID: 'organisation',
+      LABEL: "Buyer's company or organisation name",
+    },
```

Comment:
Can we use `FIELD_IDS` to define each object/field please e.g

```
[FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.COMPANY_HOUSE.INPUT]: {
  LABEL: '',
}
```

Also no need for ID in this file

Reply:
this is filed declaration in e2e we need this

Reply:
Yes, but it can come from `FIELD_IDS` instead, we currently have the country and organisation ID defined in here and `FIELD_IDS`, we used field IDs everywhere else